.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
Proponowałbym jednak wyniesienie środowiska uruchomieniowego do innego folderu:
- `docker`
- `source`

Reply:
Przeniosłem sprawy dockerowe do `docker`. Nie chcę ruszać reszty na razie, bo sam nie jestem maintainerem, a jest sporo ścieżek pohardcode'owanych. Może rzucić to jako osobny task?